sbc_compassion/views/import_letters_history_view.xml

```diff
@@ -8,29 +8,27 @@
         <field name="arch" type="xml">
             <form string="Imports">
                 <header>
-                    <field name="state" widget="statusbar" statusbar_visible="draft,open,done"/>
+                    <field name="state" widget="statusbar" statusbar_visible="draft,pending,open,done"/>
```

Comment:
I prefer to hide the pending state (except when it is in this state), it makes it more clean. So I think we can revert to as it was before.